### PR TITLE
remove lines with code line number from screen-output

### PR DESCRIPTION
**Pull request by bobmyhill**
Several of the tests which purposefully fail report line numbers of errors in the screen-output. This PR replaces those lines so that the test outputs don't need to be changed so often.

* [x] I have followed the [instructions for indenting my code](../blob/master/CONTRIBUTING.md#making-aspect-better).
* [x] I have tested my new feature locally to ensure it is correct.

### CODE REVIEW

File: tests/cmake/default [hunk 1]
```diff
@@ -31,12 +33,17 @@ while ( <STDIN> )
 	$skip=1;
     }
 
+    if ($_ =~ m/^\s*An error occurred in line/)
+    {
+  $line = "(line in output replaced by default.sh script)\n";
```
Comment (tjhei) [inline tests/cmake/default:38]: Indentation seems to be weird

Comment (bobmyhill) [reply tests/cmake/default]: The indentation of the whole file was weird. Now fixed. I don't understand why the tests failed (the script works fine on my machine). Let's see if they pass now.

**Comment by tjhei** [comment]
looks good, but you will need to update the test results from a couple other tests.

**Comment by bobmyhill** [comment]
Ok, this should now work. I've restored some of the old deleted lines, as I think the original intention was to only replace lines that printed out relative file locations, so the stored files now retain more information.

File: tests/composition_names_duplicates/screen-output [hunk 2]
```diff
@@ -15,7 +15,7 @@ Number of degrees of freedom: 49 (18+4+9+9+9)
 An exception happened on MPI process <0> while running the visualization postprocessor <N6aspect11Postprocess27VisualizationPostprocessors18MaterialPropertiesILi2EEE>: 
 
 An error occurred in file <visualization.cc> in function
-(line in output replaced by default.sh script)
+    void aspect::Postprocess::{anonymous}::add_data_names_to_set(const std::vector<std::__cxx11::basic_string<char> >&, std::set<std::__cxx11::basic_string<char> >&)
```
Comment (tjhei) [inline tests/composition_names_duplicates/screen-output:18]: the problem is that the arguments are formatted slightly differently if you run with a different compiler (for example ``__cxx11::basic_string here``).

Comment (bobmyhill) [reply tests/composition_names_duplicates/screen-output]: Oh darn. Ok, I've re-removed those lines and squashed the last two commits.

**Comment by tjhei** [comment]
Can you rebase, please?

**Pull request merged**